domain/src/DomainValidator.php

```diff
@@ -3,10 +3,11 @@
 namespace Drupal\domain;
 
 use Drupal\Core\Config\ConfigFactoryInterface;
+use Drupal\Core\Entity\EntityTypeManagerInterface;
 use Drupal\Core\Extension\ModuleHandlerInterface;
 use Drupal\Core\StringTranslation\StringTranslationTrait;
 use Drupal\Component\Utility\Unicode;
-use GuzzleHttp\ClientInterface;
```

Comment:
Interesting. I checked the Guzzle code again, and get() is a magic method that is only in the client and not in the interface. I cannot find a way to believe core isn't wrong.

One way to follow the (very desirable) practice of coding to an interface and still have code that is consistent with the interface is to forgo get() and use request('get'). I have changed the code to use request() instead of get(). Do you think that is a reasonable approach?